ospfd/ospf_ri.c

```diff
@@ -2047,6 +2095,1018 @@ DEFUN (no_pce_cap_flag,
 	return CMD_SUCCESS;
 }
 
+DEFUN (flex_algo_defn,
```

Comment:
all new cli must use defpy

Reply:
Wasn't aware of that. But I will take care of these now.